packages/expo-updates/android/src/main/java/expo/modules/updates/manifest/NewManifest.kt

```diff
@@ -53,7 +53,7 @@ class NewManifest private constructor(
       assetList.add(
         AssetEntity(
           mLaunchAsset.getString("key"),
-          mLaunchAsset.getString("contentType")
+          mLaunchAsset.getString("type")
```

Comment:
-> `fileExtension`? (+ same on iOS)

Reply:
yes! Just waiting for `www` and `spec` PR to stabilize before propagating the rename.